omnibus/package-scripts/chef-workstation/postinst

```diff
@@ -25,7 +43,25 @@ else
     PREFIX="/usr"
 fi
 
-binaries="chef-run berks chef chef-cli chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client"
+binaries="chef-run berks chef chef-cli chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client hab"
+
+old_hab_path=$(which hab)
+
+if [ "$old_hab_path" != "" ]; then
+    loc=$(readlink "$old_hab_path") # If it's not a link readlink gives back the same path.
```

Comment:
Will change this to usr PREFIX/bin/hab  instead of looking to detect it anywhere in the path.